### ISSUE TITLE
S3V4 default use streaming sign and can not change.

### ISSUE THREAD

**Issue by xiangcai1215** [minio/warp#318]
./warp put --disable-multipart --duration=1m --obj.size=10KiB --host=127.0.0.1:9500 --access-key=*** --secret-key=** --region=redoss-region1 --signature=S3V4 --insecure  --noclear --autoterm

why use "--signature=S3V4" default use stream put. but we do not support 'authTypeStreamingSigned'.

Can this provide some opts for use ’authTypeSigned‘  while not stream.

**Comment by klauspost** [comment]
Could you rephrase your question?

**Comment by T-TRz879** [comment]
Maybe he said `--disable-sha256-payload=true`

**Comment by klauspost** [comment]
Could be.
